src/appengine/handlers/cron/throttler.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2023 Google LLC
```

Comment:
nit: just put this in triage.py. 

As is, the name is a little too generic, when it's very specific to issue filing.